README.rst

```diff
@@ -35,10 +35,12 @@ PyGMT
 Disclaimer
 ----------
 
-🚨 **This package is in the early stages of design and implementation.** 🚨
+🚨 **This package is still undergoing rapid development.** 🚨
 
-All functions/classes/interfaces are subject to change as we experiment with new design
-ideas and implement new features. **This is NOT a finished product.**
+All of the API (functions/classes/interfaces) is subject to change until we reach v1.0.0
+as per the `semantic versioning specification <https://semver.org/spec/v2.0.0.html>`__.
+There may be non-backward compatible changes as we experiment with new design ideas and
+implement new features. **This is not a finished product, use with caution**
```

Comment:
:+1: nice, I like this! Though most people will ignore it and complain when we break things...